### DISCUSSION TITLE
New functional requirements of filter markers

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#73] [Ideas]
When there are many kinds of icons, it is recommended to add two functions for clear and quick selection
A. Check box: Sellect all / Desellect all
B. Marker icon

As shown in the figure below：
![A1](https://user-images.githubusercontent.com/84388916/119740950-5d631800-beb7-11eb-88bf-8cc65ceb3dca.jpg)

I feel that I have a lot of needs, which is troublesome for you. Thank you here !

**Comment by valentine195** [comment]
Added in 3.13.4

**Reply by zhubib1234** [reply]
thanks

**Reply by zhubib1234** [reply]
It's very kind of you.

**Comment by Moonbase59** [comment]
I especially love that the map marker is now shown next to its text. :+1:
